### PR TITLE
Add host survey function and clean code

### PR DESCRIPTION
**Pull request by MalwareMechanic**
Boxstarter was throwing an error because it intercepts the function named "choco". Renamed to "chocolatey" to prevent Boxstarter from intercepting.

### CODE REVIEW

**Comment by MalwareMechanic** [comment]
Added some code clean up as well to not have to further increase nuspec version

**Comment by MalwareMechanic** [comment]
Removed host survey from flarevm.installer.vm  as this is a package meant to be installed by Boxstarter. After a reboot, Boxstarter will simply rerun flarevm.installer.vm from the beginning. This means we'd fill the logs with just host survey info. I'll call this function from the installation script instead.

File: packages/common.vm/README.md [hunk 1]
```diff
@@ -31,7 +27,8 @@ The environment variables below are configurable by the user:
   - Default Path:
     - *`%SYSTEMDRIVE%`*`\Tools`
   - Use:
-    - Common directory where actual tools may be installed if not installed to the default Chocolatey package directory
+    - Directory where actual tools may be installed if not installed to the default Chocolatey package directory
 
 ## Common PowerShell Module
-We have implemented a new PowerShell module that provides commonly used functions. Any package may now make use of these common functions to reduce duplication.
+We have implemented a PowerShell module that provides commonly used functions. Any package may now make use of these common functions to reduce duplication. Module is saved to: *`%PROGRAMDATA%`*`\_VM\vm.common\vm.common.psm1`
```
Comment (mr-tz) [inline packages/common.vm/README.md:33]:
```suggestion
We have implemented a PowerShell module that provides commonly used functions. Any package may now make use of these common functions to reduce duplication. The module is saved to: *`%PROGRAMDATA%`*`\_VM\vm.common\vm.common.psm1`
```

**Review by mr-tz — APPROVED** [review]
caveat: I did not review all the code changes in detail.

**Pull request merged**